src/Components/WelcomePage/WelcomePage.js

```diff
@@ -4,60 +4,96 @@ import Dimensions from 'Dimensions';
 import { ThemeProvider, Button } from 'react-native-elements';
 import { AddUser } from '../../Actions';
 import { connect } from 'react-redux';
+import { RalewayText, RalewayBoldText } from '../../Utilities/RalewayText';
 
 const DismissKeyboard = ({ children }) => (
   <TouchableWithoutFeedback onPress={() => Keyboard.dismiss()}>
     {children}
   </TouchableWithoutFeedback>
 );
 
+let themeColor = '#667D90';
+let accentOne = '#7C9DB1';
+let accentTwo = '#ACC6D0';
+let windowHeight = Dimensions.get('window').height;
+let windowWidth = Dimensions.get('window').width;
+
 const theme = {
+
   colors: {
-    primary: '#D3E2EB',
+    primary: accentOne,
   },
+
   Button: {
-    titleStyle: {
-      color: 'black',
-    },
-  }
+    titleStyle: { color: 'white', fontFamily: 'raleway' },
+    width: Dimensions.get('window').width - 50,
+  },
+
 }
 
 class WelcomePage extends Component {
-  static navigationOptions = {
-    header: null
-  }
-
   constructor(props) {
     super(props);
     this.state = {
-      userName: ''
+      userName: '',
+      showError: false
     }
   }
 
+  static navigationOptions = {
+    header: null
+  }
+
   handleChange = (name) => {
-    this.setState({ userName: name })
+    this.setState({ 
+      userName: name, 
+      showError: false 
+    });
   }
 
   handleSave = () => {
     const { userName } = this.state;
-    this.props.AddUser(userName);
-    this.props.navigation.navigate('homePage');
+
+    if (userName) {
+      this.props.AddUser(userName);
+      this.props.navigation.navigate('homePage');
+    } else {
+      this.setState({showError: true});
+    }
+
+  }
+
+  renderSubmit = () => {
+    let buttonTitle = this.state.showError ? 'First, Enter A Name' : "Let's Get Started";
+
+    return (
+      <ThemeProvider theme={theme}>
+        {
+          this.state.userName
+            ? <Button title={buttonTitle} raised={false} onPress={this.handleSave} />
+            : <Button title={buttonTitle} raised={false} onPress={this.handleSave} />
+        }
+      </ThemeProvider>
+    )
   }
 
   render() {
     return (
       <DismissKeyboard>
         <View style={styles.background}>
-          <View>
-            <View style={styles.headerText}>
-              <Text style={styles.headerTextOne}>Welcome to</Text>
-              <Text style={styles.headerTextTwo}>Get<Text style={styles.headerTextThree}>Swole</Text></Text>
-              <Text style={styles.headerTextFour}>What should we call you?</Text>
-            </View>
-            <TextInput style={styles.textBox} onChangeText={(name) => this.handleChange(name)} placeholder=' ex. Chad...' />
-            <ThemeProvider theme={theme}>
-              <Button title="Let's get started" raised={true} onPress={this.handleSave} />
-            </ThemeProvider>
+          <View style={styles.header}>
+            <RalewayText text="Welcome To" style={styles.welcomeText} />
```

Comment:
Consider using a different welcome message.  

Glad you chose to: GET SWOLE
You're about to GET SWOLE

Just a thought